linux_os/guide/services/ntp/group.yml

```
@@ -59,6 +59,8 @@ description: |-
         {{{ weblink(link="https://help.ubuntu.com/lts/serverguide/NTP.html") }}}
     {{% elif "debian" in product %}}
         {{{ weblink(link="https://wiki.debian.org/NTP") }}}
+    {{% elif "aliyun" in product %}}
```

```suggestion
    {{% elif "alinux" in product %}}
```
The product names are [`alinux2`](https://github.com/ComplianceAsCode/content/blob/master/products/alinux3/product.yml) and [`alinux3`](https://github.com/ComplianceAsCode/content/blob/master/products/alinux3/product.yml), so this will never be true.